### PR TITLE
Initial AI voice transform service with ye200k model 

### PR DESCRIPTION
**Pull request by SamYu**
Defaults to ye200k model and config (model file not committed due to size)

### CODE REVIEW

File: configs/config.json
Comment (lnittman) [inline configs/config.json:1]: anything in here that we'd look to tweak going forwards?

**Review by lnittman — APPROVED** [review]

File: stems_server.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    out_wav_path = io.BytesIO()
+    soundfile.write(out_wav_path, audio,  svc_model.target_sample, format="wav")
+    out_wav_path.seek(0)
+    output_name = f'${audio_url_filename}_transformed.wav'
+    return send_file(out_wav_path, download_name=output_name, as_attachment=True)
+
+
+if __name__ == '__main__':
+    # each model and config are corresponding
+    model_name = "logs/44k/ye200k/G_199200.pth"
```
[the remark this answers is not in the document]
Comment (SamYu) [reply stems_server.py:95]: They recommend putting the pretrained models in the `logs/44k` folder, not sure why its named that way. When we add more I can think of a better naming scheme for the actual model files themselves, maybe just so we only need to map between the folder name and the requested AI voice.

File: configs/config.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
```
[the remark this answers is not in the document]
Comment (SamYu) [reply configs/config.json:1]: These are given by the pretrained model so I doubt we'll need to tweak them unless we start creating our own models. I will come up with a better folder structure for these so we know which config is for which AI voice model. 😁

File: stems_server.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        "cr_threshold" : cr_threshold,
+        "k_step": k_step
+    }
+    audio = svc_model.slice_inference(**kwarg)
+
+    # return the processed audio as a downloadable file
+    out_wav_path = io.BytesIO()
+    soundfile.write(out_wav_path, audio,  svc_model.target_sample, format="wav")
+    out_wav_path.seek(0)
+    output_name = f'${audio_url_filename}_transformed.wav'
```
[the remark this answers is not in the document]
Comment (SamYu) [reply stems_server.py:89]: Thats a good point, also we may need to consider if multiple people are transforming the same stem at the same time, would the file names clash with each other? Probably need to consider storage as well and how long we want to keep these files before deletion.

**Pull request merged**